src/cpp/ext/otel/otel_plugin.h

```diff
@@ -365,11 +395,10 @@ class OpenTelemetryPlugin : public grpc_core::StatsPlugin {
       grpc_core::GlobalInstrumentsRegistry::GlobalDoubleGaugeHandle /*handle*/,
       double /*value*/, absl::Span<const absl::string_view> /*label_values*/,
       absl::Span<const absl::string_view> /*optional_values*/) override {}
-  // TODO(yashkt, yijiem): implement async instrument.
-  void AddCallback(grpc_core::RegisteredMetricCallback* /*callback*/) override {
-  }
-  void RemoveCallback(
-      grpc_core::RegisteredMetricCallback* /*callback*/) override {}
+  void AddCallback(grpc_core::RegisteredMetricCallback* /*callback*/)
```

Comment:
Parameter name shouldn't be commented out anymore for these two methods.

Reply:
done.